src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml

```diff
@@ -724,68 +757,445 @@ tests:
+          [1722930734.451] [29970:29970] [-] 2 times the momentary switch has been pressed
+          [1722930734.451] [29970:29970] [ZCL] SwitchServer: OnMultiPressOngoing
+
+
+          ./chip-tool switch read-event multi-press-ongoing 1 1
+
+          Verify MultiPressOngoing event with NewPosition set to 1 and CurrentNumberOfPressesCounted is 2 on DUT(Chip-tool) log:
+
+          [1722930652.716] [31360:31362] [DMG]         SuppressResponse = true,
+          [1722930652.716] [31360:31362] [DMG]         InteractionModelRevision = 11
+          [1722930652.716] [31360:31362] [DMG] }
+          [1722930652.717] [31360:31362] [TOO] Endpoint: 1 Cluster: 0x0000_003B Event 0x0000_0005
+          [1722930652.717] [31360:31362] [TOO]   Event number: 65565
+          [1722930652.717] [31360:31362] [TOO]   Priority: Info
+          [1722930652.717] [31360:31362] [TOO]   Timestamp: 1722930534106
+          [1722930652.717] [31360:31362] [TOO]   MultiPressOngoing: {
+          [1722930652.717] [31360:31362] [TOO]     NewPosition: 1
+          [1722930652.717] [31360:31362] [TOO]     CurrentNumberOfPressesCounted: 2
+          [1722930652.717] [31360:31362] [TOO]    }
+
+
+          wait 0.2 seconds
+
+          echo '{"Name":"ShortRelease","PreviousPosition":1}' > /tmp/chip_all_clusters_fifo_<PID>
+
+          Verify ShortRelease event with PreviousPosition set to 1 on TH(all-cluster-app) log:
+
+          [1722930799.722] [29970:29972] [-] Received payload: "{"Name":"ShortRelease","PreviousPosition":1}"
+          [1722930799.723] [29970:29970] [DMG] Endpoint 1, Cluster 0x0000_003B update version to 6903ad3f
+          [1722930799.723] [29970:29970] [-] The the previous value of the CurrentPosition when the momentary switch has been released:1
+          [1722930799.723] [29970:29970] [ZCL] SwitchServer: OnShortRelease
+
+
+          ./chip-tool switch read-event short-release  1 1
+
+          Verify ShortRelease event with PreviousPosition set to 1 on DUT(chip-tool) log:
+
+          [1722930819.997] [31413:31415] [DMG]         SuppressResponse = true,
+          [1722930819.997] [31413:31415] [DMG]         InteractionModelRevision = 11
+          [1722930820.000] [31413:31415] [TOO] Endpoint: 1 Cluster: 0x0000_003B Event 0x0000_0003
+          [1722930820.000] [31413:31415] [TOO]   Event number: 65567
+          [1722930820.000] [31413:31415] [TOO]   Priority: Info
+          [1722930820.000] [31413:31415] [TOO]   Timestamp: 1722930799723
+          [1722930820.001] [31413:31415] [TOO]   ShortRelease: {
+          [1722930820.001] [31413:31415] [TOO]     PreviousPosition: 1
+          [1722930820.001] [31413:31415] [TOO]    }
+
+
+          wait 2 seconds
+
+          echo '{"Name":"MultiPressComplete","PreviousPosition":1,"TotalNumberOfPressesCounted":2}' > /tmp/chip_all_clusters_fifo_<PID>
+
+          Verify MultiPressComplete event with PreviousPosition set to 1 AND TotalNumberOfPressesCounted is 2  on TH(all-cluster-app) log:
```

Comment:
This whole test has so many manual steps, it is unverifiable.

Reply:
this is a client test case, most of the client test cases are manual for now.